tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +111,19 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory  
+    assert item_b in tai.inventory  
+    assert item_f in tai.inventory  
+    assert item_d in jesse.inventory  
+    assert item_e in jesse.inventory 
+    assert item_c in jesse.inventory  
+    assert item_c not in tai.inventory  
```

Comment:
If you're going to assert that item c is not in tai's inventory, I would also asset that item f is not in jesse's inventory! Just in case!